package.json

```diff
@@ -54,6 +45,7 @@
   },
   "main": "d2l-offscreen.js",
   "dependencies": {
-    "@polymer/polymer": "^3.0.0"
+    "@brightspace-ui/core": "^1.9.1",
```

Comment:
We need to be explicitly calling out the version of core that's required here ^. Otherwise semver can resolve to something lower, which won't have offscreen in core and it'll fail. I'll be updating some of the others (link, localize-behavior, etc.).